canonical-websites/www.ubuntu.com
# new /desktop/organisations page - 18.04

## Done

Build new organisation page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: <http://0.0.0.0:8001/desktop/organisations>
- Check page matches [the copy doc](https://docs.google.com/document/d/1q3_fQB3PUGbREsbCC4FnmHrpgn9nIWtMwyhDC6CXhNE/edit#) and [the design](https://user-images.githubusercontent.com/36884067/38875497-65deee80-4252-11e8-997d-01adb6b310b3.jpg)


## Issue / Card

Fixes [#470](https://github.com/ubuntudesign/web-squad/issues/470)
